anvil/src/udev.rs

```diff
@@ -893,17 +916,17 @@ fn render_surface<'a>(
         {
             surface.fps_element.update_fps(surface.fps.avg().round() as u32);
             surface.fps.tick();
+            elements.push(CustomRenderElements::Fps(surface.fps_element.clone()));
         }
     }
 
     // and draw to our buffer
     // TODO we can pass the damage rectangles inside a AtomicCommitRequest
     #[cfg(feature = "debug")]
```

Comment:
This is probably some left-over, both `cfg` parts do the same.

Reply:
fixed